src/rpc/message.cpp

```diff
@@ -174,30 +118,15 @@ rapidjson::Value& FullMessage::getMessage()
 
 rapidjson::Value FullMessage::getMessageCopy()
```

Comment:
I'm a bit worried about this method. It does return a copy, as promised, but it's still backed by the `doc` member we have. This means that if the `FullMessage` is destructed after obtaining the copy, we'd end up reading deallocated memory, resulting in undefined behaviour.

Since this method comes with some gotchas, I'd say we should either remove the method (it's not used anywhere), or add a clear warning to the method to avoid running into problems later on.

Reply:
It "ok" for now, but this method probably needs to be punted as you suggest. I haven't thought about how to re-arrange this yet. Do this now too or ... ? The relevant code has been in several releases, fwiw.

Reply:
Read fail. I responded too quickly before reading the unused portion. I will chop this function then.